### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by huzzy993**

### CODE REVIEW

File: src/main/java/com/factly/dega/domain/Factcheck.java [hunk 1]
```diff
@@ -35,12 +35,14 @@
     @Field("introduction")
     private String introduction;
 
+    @NotNull
```
Comment (ntallapa12) [inline src/main/java/com/factly/dega/domain/Factcheck.java:38]: @deshetti is it ok of its not null

File: src/main/java/com/factly/dega/domain/Factcheck.java [hunk 2]
```diff
@@ -35,12 +35,14 @@
...
     @Field("summary")
     private String summary;
 
     @Field("excerpt")
     private String excerpt;
 
+    @NotNull
```
Comment (ntallapa12) [inline src/main/java/com/factly/dega/domain/Factcheck.java]: factcheck in draft mode does not have this date. not null should be removed

File: src/main/java/com/factly/dega/domain/Factcheck.java (hunk 1 above)
Comment (deshetti) [reply src/main/java/com/factly/dega/domain/Factcheck.java]: changed it to NotNull

Comment (deshetti) [reply src/main/java/com/factly/dega/domain/Factcheck.java:38]: Yes, currently summary is not null

File: src/main/java/com/factly/dega/domain/Factcheck.java (hunk 2 above)
Comment (deshetti) [reply src/main/java/com/factly/dega/domain/Factcheck.java]: yes, published_date is nullable. this needs to be changed.

**Review by deshetti — APPROVED** [review]

**Pull request merged**